src/microbe_stage/Microbe.cs

```diff
@@ -1295,15 +1297,18 @@ private void HandleOsmoregulation(float delta)
     /// </summary>
     private void ApplyATPDamage()
     {
-        if (Compounds.GetCompoundAmount("atp") <= 0.0f)
+        if (noATPDamageWait >= Constants.NO_ATP_DAMAGE_TIME)
```

Comment:
This is not what I had in mind. The actual cause why the glucose drains immediately and the player runs out of ATP really fast needs to be fixed.